src/CounterInterceptor/ClientCounter.cs

```diff
@@ -0,0 +1,87 @@
+namespace TemporalioSamples.CounterInterceptor;
+public record ClientCounts
+{
+    public uint Executions { get; internal set; }
+
+    public uint Signals { get; internal set; }
+
+    public uint Queries { get; internal set; }
+
+    public override string ToString()
+    {
+        return
+                "\n\tTotal Number of Workflow Exec: " + Executions +
+                "\n\tTotal Number of Signals: " + Signals +
+                "\n\tTotal Number of Queries: " + Queries;
+    }
```

Comment:
```suggestion
    public override string ToString() =>
        $"\n\tTotal Number of Workflow Exec: {Executions}\n\t" +
        $"Total Number of Signals: {Signals}\n\tTotal Number of Queries: {Queries}";
    }
```

Can break up long line, but might as well use interpolation, and might as well use `=>` for single-line methods. Granted, despite what I said before, using `ToString()` for multiline strings only for a specific scenario is a bit much, but fine.

Reply:
Done